Docs/source/running_cpp/parameters.rst

```diff
@@ -1236,7 +1238,10 @@ Numerics and algorithms
 * ``macroscopic.sigma``, ``macroscopic.epsilon``, ``macroscopic.mu`` (`double`)
     To initialize a constant conductivity, permittivity, and permeability of the
     computational medium, respectively. The default values are the corresponding values
-    in vacuum.
+    in vacuum. 
+    In particular, if `USE_LLG=TRUE` in the GNUMakefile and `mag_Ms` is non-zero, ``macroscopic.mu`` must be set to the vacuum value (``mu0``, i.e. 1.25663706212e-06). 
```

Comment:
Are we actually enforcing this in the code with an Abort(), or warning with a Warn(), or neither?  Is it more accurate to say that the value of mu_r in regions where Ms != 0 simply doesn't matter?

Reply:
Operationally, it is true that the value of mu_r in regions where Ms != 0 simply doesn't matter. But I'm concerned this might be misleading. Maybe a warning should be added in the next PR (together with this document)?